src/Arcus.Observability.Tests.Integration/Serilog/ApplicationInsightsSinkTests.cs

```diff
@@ -3,28 +3,38 @@
 using System.Net;
 using System.Net.Http;
 using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
 using Arcus.Observability.Correlation;
 using Arcus.Observability.Telemetry.Serilog.Enrichers;
 using Arcus.Observability.Tests.Core;
 using Bogus;
 using Microsoft.AspNetCore.Http;
+using Microsoft.Azure.ApplicationInsights;
+using Microsoft.Azure.ApplicationInsights.Models;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Logging;
+using Microsoft.Rest;
 using Moq;
+using Polly;
+using Polly.Timeout;
 using Serilog;
 using Serilog.Configuration;
 using Serilog.Core;
 using Serilog.Events;
 using Serilog.Extensions.Logging;
 using Xunit;
 using Xunit.Abstractions;
+using Xunit.Sdk;
 using ILogger = Microsoft.Extensions.Logging.ILogger;
 
 namespace Arcus.Observability.Tests.Integration.Serilog
 {
     [Trait(name: "Category", value: "Integration")]
     public class ApplicationInsightsSinkTests : IntegrationTest
```

Comment:
This looks good to me, I'm just wondering if we should include correlation to ensure that it's the specific entry we want to have, but I'll leave it up to you to judge.

Reply:
By using generated values, and including them in the assert, we're pretty sure that it's the correct one we're after. Several of them also include a time range of 1h to limit this even further. The correletion we could add will probably not be available in the retrieved entries anyway bc they are custom properties.

Reply:
Sounds good to me, but having dedicated test for the correlation enricher to verify it works might be interesting, no?

Just thinking out loud.

Reply:
Moved to #93 where we can talk about how.